spec/services/encrypted_key_maker_spec.rb

```diff
@@ -57,6 +61,16 @@
 
         expect(subject.unlock(user_access_key, encryption_key)).to eq hash_E
       end
+
+      it 'raises exception when params are invalid' do
+        expect do
+          subject.unlock(user_access_key, '!@#$%^&*()')
+        end.to raise_error Pii::EncryptionError
+
+        expect do
+          subject.unlock('foobar', 'zzzz')
+        end.to raise_error Pii::EncryptionError
```

Comment:
I've always used brackets. I didn't realize we had that many `do end` instances. I personally prefer brackets when chaining methods, but I'm open to `do end` if there's consensus.